src/core/execute.c

```diff
@@ -948,8 +949,13 @@ static int setup_pam(
                  * and this will make PR_SET_PDEATHSIG work in most cases.
                  * If this fails, ignore the error - but expect sd-pam threads
                  * to fail to exit normally */
+
+                if (setgroups(0, NULL) < 0)
+                        log_warning_errno(errno, "Failed to setgroups() in sd-pam: %m");
+                if (setresgid(gid, gid, gid) < 0)
+                        log_warning_errno(errno, "Failed to setresgid() in sd-pam: %m");
```

Comment:
oh, sorry, I mean there is a little difference. This doesn't really matter. Thanks!